views/pages/organisation/index.html

```diff
@@ -17,25 +17,36 @@
     {% block mainContent %}
 
     <div class="govuk-grid-column-two-thirds">
+
       {% set status = '' %}
       {% set html = '' %}
 
-      {% if addTeamAttempt %}
-        {% if addTeamSuccess %}
-          {% set status = "success" %}
-          {% set html = "<h2>You have successfully added team " + addedTeamName + "</h2>" %}
-        {% else %}
-          {% set status = "failure" %}
-          {% if teamExistsError %}
-            {% set html = "<h2>Unable to add team as " + addedTeamName + " already exists in organisation</h2>" %}
+      {% if addTeamAttempt or editTeamAttempt %}
```

Comment:
this section seems soo confusing to be honest had to figure out if and endif are relevant for the block but seems all okay

Reply:
This should be static now, so I'm going to leave it...if we have to add more, then I think perhaps splitting it all could be done then.